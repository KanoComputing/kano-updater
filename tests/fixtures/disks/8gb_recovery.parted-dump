Model: SD SS08G (sd/mmc)
Disk /dev/mmcblk0: 15523840s
Sector size (logical/physical): 512B/512B
Partition Table: msdos
Disk Flags: 

Number  Start     End        Size       Type      File system  Flags
 1      8192s     3386718s   3378527s   primary   fat32        lba
 2      3386719s  15523839s  12137121s  extended
 5      3391488s  3457021s   65534s     logical   ext4
 6      3457024s  3866621s   409598s    logical   fat16        lba
 7      3866624s  15523839s  11657216s  logical   ext4

