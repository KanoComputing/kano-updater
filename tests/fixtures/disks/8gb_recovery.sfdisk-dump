{
   "partitiontable": {
      "label": "dos",
      "id": "0x000198c8",
      "device": "/dev/mmcblk0",
      "unit": "sectors",
      "partitions": [
         {"node": "/dev/mmcblk0p1", "start": 8192, "size": 3378527, "type": "e"},
         {"node": "/dev/mmcblk0p2", "start": 3386719, "size": 12137121, "type": "5"},
         {"node": "/dev/mmcblk0p5", "start": 3391488, "size": 65534, "type": "83"},
         {"node": "/dev/mmcblk0p6", "start": 3457024, "size": 409598, "type": "c"},
         {"node": "/dev/mmcblk0p7", "start": 3866624, "size": 11657216, "type": "83"}
      ]
   }
}
