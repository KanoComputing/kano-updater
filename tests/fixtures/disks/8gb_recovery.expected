{
    "commands": [
        "parted /dev/mmcblk0 --script unit % resizepart 2 100 || parted /dev/mmcblk0 unit % resizepart 2 Yes 100",
        "parted /dev/mmcblk0 --script unit % resizepart 7 100 || parted /dev/mmcblk0 unit % resizepart 7 Yes 100",
        "partprobe /dev/mmcblk0",
        "resize2fs /dev/mmcblk0p7"
    ]
}
